reference/resources/payment_method.yaml

```
@@ -104,6 +104,15 @@ properties:
     maxLength: 5
     nullable: true
 
+  approval_target:
+    type: string
+    example: blank
+    nullable: true
+    description: The optional target than an approval URL must be opened in.
+      If not provided, there is no specific requirement.
+    enum:
+      - blank
```

We seem to re-use this prop all over. Might be easier to make a ref.

For example

```suggestion
  approval_target:
    $ref: "../properties/approval_target.yaml"
```